sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/query.sql

```diff
@@ -171,31 +111,35 @@ WITH unioned AS (
 ),
 search_clients AS (
   SELECT
-    *
+    client_id,
+    submission_date,
+    ad_click,
+    organic,
+    search_count,
+    search_with_ads
   FROM
     search_derived.mobile_search_clients_daily_v1
   WHERE
-    submission_date = DATE_ADD(@submission_date, INTERVAL 1 DAY)
+    submission_date = @submission_date
```

Comment:
I noticed the different [search pings](https://mozilla.github.io/glean/book/user/pings/metrics.html#scheduling) sent, which indicate the 1 day difference between main ping sent today and search ping sent the following day at 4:00 for most cases.
In the discussion you link the  call for a `baseline_clients_last_seen` LEFT JOIN `metrics_clients_last_seen`, however `unified_metrics` uses `mobile_search_clients_daily_v1` LEFT JOIN `[clients_last_seen_joined](https://github.com/mozilla/bigquery-etl/blob/main/sql_generators/glean_usage/templates/clients_last_seen_joined.query.sql)` and this last table has already applied ` WHERE submission_date = DATE_ADD(@submission_date, INTERVAL 1 DAY)`. It seems like a duplication, so it wouldn't be required here. Or perhaps we need to double check the tables used?

Reply:
I think your analysis here is correct. I just wanted to make sure this change was intentional and that you had relevant context.

And after looking at this I suspect `clients_last_seen_joined` as currently set up may be incorrect, looking for metrics data that has not yet been populated. I wrote up https://bugzilla.mozilla.org/show_bug.cgi?id=1769109 to investigate.

Reply:
So it would need to query one day before, to get the full dataset,
Which of course implies that data would be updated until 1 day before.

`WITH baseline AS (
  SELECT 
    *
  FROM 
    `{{ project_id }}.{{ app_name }}.baseline_clients_last_seen` 
  WHERE 
    submission_date = DATE_SUB(@submission_date, INTERVAL 1 DAY)
),
metrics AS (
  SELECT 
    * 
  FROM 
    `{{ project_id }}.{{ app_name }}.metrics_clients_last_seen`
  WHERE 
    submission_date = @submission_date
)`

Reply:
> But that would imply that data would have a 1 day delay in comparison to how it is updated now

Yes, that is a downside of the one-day skew called out in the proposal. We will need to accept that and communicate that to users in order to have the `metrics` ping-based fields available in this table.